src/components/button-toggle/button-toggle.style.ts

```diff
@@ -106,26 +138,49 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
     color: var(--colorsActionMinor500);
   }
 
-  ${({ buttonIcon, buttonIconSize, size }) =>
+  ${({ buttonIcon, buttonIconSize, size, variant }) =>
     buttonIcon &&
     buttonIconSize === "large" &&
     css`
       height: ${heightLargeIconConfig[size]}px;
-      padding: 0 ${paddingLargeIconConfig[size]}px;
+      padding: ${variant === "default"
+        ? `0 ${paddingLargeIconConfig[size]}px;`
+        : `var(--spacing000) var(--spacing100) var(--spacing000) var(--spacing100)`};
       flex-direction: column;
     `}
-  ${({ disabled }) =>
+
+  ${({ disabled, variant }) =>
```

Comment:
you'll need to destructure size here

Reply:
It would probably be good to look through this file and consolidate it a bit as it there's multiple points where we are destructuring props (for example `size` and `variant`) and then later in the file doing the same thing which makes it more difficult to read maintain imo. If/ when you do be careful with the ordering as we will need the `minor` styling to override the default in most cases